README.md

```diff
@@ -1,32 +1,355 @@
-# Lesson Title
+# Modifying Objects
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Review the concept of reference types
+- Practice cloning objects
+- Practice modifying objects
+- Write pure functions that act on objects
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+So far we've learned how to access information from objects, both directly using
+dot or bracket notation, and dynamically using a `for...in` loop. In this
+lesson, we'll learn how to copy and modify objects. We'll also learn how to
+modify objects _nondestructively_, so we can keep our functions pure.
 
-## Topic 1
+## Review of Reference Types
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+You may recall that we learned in a previous lesson that arrays are what is
```

Comment:
Maybe simplify the first portion to just:

You may recall that arrays are _reference_ types ...